neo4j-ruby-driver.gemspec

```diff
@@ -33,22 +36,24 @@ Gem::Specification.new do |spec|
     `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   end
 
-  ffi = ENV['SEABOLT_LIB']&.length&.positive?
-  pdir = ffi ? 'ffi' : 'jruby'
+  jruby = RUBY_PLATFORM.match?(/java/)
+  pdir = if ENV.key?('SEABOLT_LIB')
+           'ffi'
+         else
+           jruby ? 'jruby' : 'ffi'
```

Comment:
this is still incorrect. ffi should be supported on jruby as well in which case the SEABOLT_LIB is required. The last `ffi` is incorrect as this case in supported (not jruby and not seabolt).
Even without understanding anything about the domain any construct like `a ? b : c ? d : b` should always raise a red flag. The conditionals should be built in a way that `b` does not have to be repeated. Following this coding standard helps detecting potential issues.